src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -155,17 +155,40 @@ export const evidenceTypeHelp = (
   </AdditionalInfo>
 );
 
+const capitalizeEachWord = (name) => {
+  const splitName = name.split(' ');
+  const capitalizedsplitName = splitName.map(word => {
+    const capFirstLetter = word[0].toUpperCase();
+    return `${capFirstLetter}${word.slice(1)}`;
+  });
+  return capitalizedsplitName.join(' ');
+};
+
+/**
+ * 
+ * @param {string} name the lower-case name of a disability
+ * @param {number} code the diagnosticCode of a disability
+ */
+export const getDisabilityName = (name, code) => {
+  if (name) {
+    return capitalizeEachWord(name);
+  } else if (code) {
+    return (getDiagnosticCodeName(code));
```

Comment:
I think we can't have this at all, unfortunately. It gave back completely wrong condition names in UAT.